assets/css/v2/bus_eink/departures/row.scss

```diff
@@ -0,0 +1,37 @@
+.departure-row {
+  padding-bottom: 30px;
+  position: relative;
+
+  &::after {
```

Comment:
I'll follow up on this with Adam (since it's inconsistent with the Bus Shelter screen), but that's how it is in the [Abstract collection](https://app.abstract.com/projects/c04b5940-4805-11e8-8262-510af7fd49fe/branches/01a916c1-28e6-4c43-9872-ac94e65c63e5/commits/latest/files/19842ee7-9b99-430a-9e0f-569ede8be4bb/layers/9F54FB0B-040D-4127-8852-AD1B22EEED97?collectionId=888b8a7f-a57f-441c-aeee-1c432e9197fe&collectionLayerId=5da3ed6f-ad23-4005-b713-0a8ca9f24489).